src/main/java/core/basesyntax/Plane.java

```diff
@@ -8,4 +8,80 @@
 
 public class Plane {
 
+    private Integer power;
+    private Integer size;
+    private String color;
+    private Integer seats;
+    private String model;
+
+    public Plane(PlaneBuilder builderForPlane) {
```

Comment:
we should have access for this constructor only within this class

Reply:
Yup, sorry i had watched wrong realizetion of Builder pathern